### PR TITLE
[New] Add jsfx@1.2.0 w/ npm auto-update

### LINKED ISSUE #11179: [Request] Add jsfx
**Library name:** jsfx
**Git repository url:** https://github.com/loov/jsfx
**npm package name or url** (if there is one): https://www.npmjs.com/package/loov-jsfx
**License (List them all if it's multiple):** MIT

### PR DESCRIPTION
**Pull request by qwe19272375**
Pull request for issue: #11179 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/loov/jsfx
 * Official website (optional, not the repository): http://loov.io/jsfx/
 * NPM package url (optional): https://www.npmjs.com/package/loov-jsfx
 * License and its reference: MIT, [ref](https://github.com/loov/jsfx/blob/master/LICENSE-MIT)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 20
   - Count of stars: 471
   - Count of forks: 38
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 1
   - Downloads in the last month: 13

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #11179, cc @loov

### CODE REVIEW

**Comment by qwe19272375** [comment]
Manually add old versions of jsfx due to different file structure.

**Comment by qwe19272375** [comment]
@cdnjs/intern3 Please help me review this PR, thanks!

File: ajax/libs/jsfx/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  "homepage": "http://loov.io/jsfx",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/loov/jsfx.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jsfx.js",
+          "jsfx.min.js"
```
Comment (extend1994) [inline ajax/libs/jsfx/package.json]: It's more flexible to use `jsfx*` instead of `jsfx.js` and `jsfx.min.js` for future maintainance.

**Comment by qwe19272375** [comment]
Use `jsfx*` get `jsfx.png` at `v1.1.0` and I'm kind of sure it should not be included.
I will discuss with author which files we have to host before `v1.1.0`

**Comment by extend1994** [comment]
@qwe19272375 Sorry about not noticing that. Then let's use `jsfx*.+(js|css|map)`.

**Comment by qwe19272375** [comment]
@cdnjs/intern3 
[loov/jsfx#18](https://github.com/loov/jsfx/issues/18) author suggested us to host versions after v1.0 and we can do it by switching to npm auto-update.
Should we use npm auto-update instead?

**Comment by qwe19272375** [comment]
As the author suggest, we should use source from npm.

**Comment by sufuf3** [comment]
I think too.
use npm is better.

**Comment by qwe19272375** [comment]
Switched to npm auto-update.
@cdnjs/intern3 please help me review it, thank you.

**Comment by qwe19272375** [comment]
ping @pvnr0082t

**Comment by sufuf3** [comment]
@qwe19272375 please help fix this PR, thank you.

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Pull request merged**